Compilation from source broken in nu-utils

### Describe the bug

When compiling `nu` from source with `cargo install nu --features dataframe`, a type error is returned as shown in snippet below.

### How to reproduce

1. Run `cargo install nu --features dataframe`.

### Expected behavior

Successful compilation of `nu.exe`.

### Screenshots

```
error[E0308]: mismatched types
  --> ...\.cargo\registry\src\github.com-1ecc6299db9ec823\nu-utils-0.83.1\src\deansi.rs:16:16
   |
16 |         if let Ok(stripped) = strip_ansi_escapes::strip(string) {
   |                ^^^^^^^^^^^^   --------------------------------- this expression has type `Vec<u8>`
   |                |
   |                expected `Vec<u8>`, found `Result<_, _>`
   |
   = note: expected struct `Vec<u8>`
                found enum `Result<_, _>`

error[E0308]: mismatched types
  --> ...\.cargo\registry\src\github.com-1ecc6299db9ec823\nu-utils-0.83.1\src\deansi.rs:37:12
   |
37 |     if let Ok(stripped) = strip_ansi_escapes::strip(string) {
   |            ^^^^^^^^^^^^   --------------------------------- this expression has type `Vec<u8>`
   |            |
   |            expected `Vec<u8>`, found `Result<_, _>`
   |
   = note: expected struct `Vec<u8>`
                found enum `Result<_, _>`

error[E0308]: mismatched types
  --> ...\.cargo\registry\src\github.com-1ecc6299db9ec823\nu-utils-0.83.1\src\deansi.rs:63:16
   |
63 |         if let Ok(stripped) = strip_ansi_escapes::strip(&string) {
   |                ^^^^^^^^^^^^   ---------------------------------- this expression has type `Vec<u8>`
   |                |
   |                expected `Vec<u8>`, found `Result<_, _>`
   |
   = note: expected struct `Vec<u8>`
                found enum `Result<_, _>`

error[E0308]: mismatched types
  --> ...\.cargo\registry\src\github.com-1ecc6299db9ec823\nu-utils-0.83.1\src\deansi.rs:84:12
   |
84 |     if let Ok(stripped) = strip_ansi_escapes::strip(&string) {
   |            ^^^^^^^^^^^^   ---------------------------------- this expression has type `Vec<u8>`
   |            |
   |            expected `Vec<u8>`, found `Result<_, _>`
   |
   = note: expected struct `Vec<u8>`
                found enum `Result<_, _>`
```

### Configuration

| key                | value                                                                                                                                                           |
| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| version            | 0.81.0                                                                                                                                                          |
| branch             |                                                                                                                                                                 |
| commit_hash        |                                                                                                                                                                 |
| build_os           | windows-x86_64                                                                                                                                                  |
| build_target       | x86_64-pc-windows-msvc                                                                                                                                          |
| rust_version       | rustc 1.69.0 (84c898d65 2023-04-16)                                                                                                                             |
| rust_channel       | stable-x86_64-pc-windows-msvc                                                                                                                                   |
| cargo_version      | cargo 1.69.0 (6e9a83356 2023-04-12)                                                                                                                             |
| build_time         | 2023-06-18 10:34:38 -07:00                                                                                                                                      |
| build_rust_channel | release                                                                                                                                                         |
| features           | dataframe, default, sqlite, trash, which, zip                                                                                                                   |
| installed_plugins  | [[redacted]] |

### Additional context

Root cause appears to be https://github.com/luser/strip-ansi-escapes/issues/17.

--- a/crates/nu-explore/src/views/record/tablew.rs
+++ b/crates/nu-explore/src/views/record/tablew.rs
@@ -772,10 +772,9 @@ fn render_column(
 }
 
 fn strip_string(text: &str) -> String {
-    strip_ansi_escapes::strip(text)
-        .ok()
-        .and_then(|s| String::from_utf8(s).ok())
-        .unwrap_or_else(|| text.to_owned())
+    String::from_utf8(strip_ansi_escapes::strip(text))
+        .map_err(|_| ())
+        .unwrap_or_else(|_| text.to_owned())
 }
 
 fn head_row_text(head: &str, style_computer: &StyleComputer) -> NuText {

--- a/crates/nu-utils/src/deansi.rs
+++ b/crates/nu-utils/src/deansi.rs
@@ -13,10 +13,8 @@ pub fn strip_ansi_unlikely(string: &str) -> Cow<str> {
     // which will be stripped. Includes the primary start of ANSI sequences ESC
     // (0x1B = decimal 27)
     if string.bytes().any(|x| matches!(x, 0..=9 | 11..=31)) {
-        if let Ok(stripped) = strip_ansi_escapes::strip(string) {
-            if let Ok(new_string) = String::from_utf8(stripped) {
-                return Cow::Owned(new_string);
-            }
+        if let Ok(stripped) = String::from_utf8(strip_ansi_escapes::strip(string)) {
+            return Cow::Owned(stripped);
         }
     }
     // Else case includes failures to parse!
@@ -34,10 +32,8 @@ pub fn strip_ansi_likely(string: &str) -> Cow<str> {
     // Check if any ascii control character except LF(0x0A = 10) is present,
     // which will be stripped. Includes the primary start of ANSI sequences ESC
     // (0x1B = decimal 27)
-    if let Ok(stripped) = strip_ansi_escapes::strip(string) {
-        if let Ok(new_string) = String::from_utf8(stripped) {
-            return Cow::Owned(new_string);
-        }
+    if let Ok(stripped) = String::from_utf8(strip_ansi_escapes::strip(string)) {
+        return Cow::Owned(stripped);
     }
     // Else case includes failures to parse!
     Cow::Borrowed(string)
@@ -60,10 +56,8 @@ pub fn strip_ansi_string_unlikely(string: String) -> String {
         .bytes()
         .any(|x| matches!(x, 0..=9 | 11..=31))
     {
-        if let Ok(stripped) = strip_ansi_escapes::strip(&string) {
-            if let Ok(new_string) = String::from_utf8(stripped) {
-                return new_string;
-            }
+        if let Ok(stripped) = String::from_utf8(strip_ansi_escapes::strip(&string)) {
+            return stripped;
         }
     }
     // Else case includes failures to parse!
@@ -81,10 +75,8 @@ pub fn strip_ansi_string_likely(string: String) -> String {
     // Check if any ascii control character except LF(0x0A = 10) is present,
     // which will be stripped. Includes the primary start of ANSI sequences ESC
     // (0x1B = decimal 27)
-    if let Ok(stripped) = strip_ansi_escapes::strip(&string) {
-        if let Ok(new_string) = String::from_utf8(stripped) {
-            return new_string;
-        }
+    if let Ok(stripped) = String::from_utf8(strip_ansi_escapes::strip(&string)) {
+        return stripped;
     }
     // Else case includes failures to parse!
     string

